CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ look through the [documentation](https://www.slicer.org/wiki/Documentation/Night
 do better.
 
   * Ask a question on the [Slicer forum](https://discourse.slicer.org)
-  * Submit a feature request or bug, or add to the discussion on the [Slicer issue tracker](https://issues.slicer.org/)
```

Comment:
In  documentation, I suggest we keep `https://issues.slicer.org/`

Reply:
@jcfr Did you still want this changed? I see you already merged this PR.

Reply:
That would be nice. 

Also,  would be nice to add the following at the end of the document:

```
[slicer-forum]: https://discourse.slicer.org
[slicer-issues]: https://issues.slicer.org
...
```

Then, update usage like this:

`[Slicer forum](https://discourse.slicer.org)`
->
`[Slicer forum][slicer-forum]`